packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -241,7 +241,7 @@ function useBlockEditorSettings( settings, postType, postId ) {
 					BLOCK_EDITOR_SETTINGS.includes( key )
 				)
 			),
-			__experimentalStyles: styles,
+			globalStyles: styles,
```

Comment:
If we're using the `globalStyles` name here, I wonder if it might also be good to make the rest of the code follow the same naming - `const { globalStyles } = useGlobalStyles();`?

Reply:
The other feedback on the other PR was making it a private setting, there are some examples of private settings keys (using Symbols, it works similarly to the private slots we worked on a while back) if we still want to do that:
https://github.com/WordPress/gutenberg/blob/a2a8ec9d6111f3630b00c7fc4b55094f7ea57b19/packages/block-editor/src/store/private-keys.js#L1-L2

https://github.com/WordPress/gutenberg/blob/a2a8ec9d6111f3630b00c7fc4b55094f7ea57b19/packages/editor/src/components/provider/use-block-editor-settings.js#L270-L275

Reply:
> I did wonder about this and thought maybe it was blurring the lines further with useGlobalStyle. I can update it.

There's already a `useGlobalStyles` as well. 🤔

Reply:
Another possibility could be `useGlobalStylesData`. It reflects a little better that these styles are the uncompiled global styles.

Reply:
That may then raise the question of whether the `globalStyles` setting should be `globalStylesData`. Another reason to make it private.

Reply:
I've made the suggested tweaks in https://github.com/WordPress/gutenberg/pull/61022/commits/c4e3b0b9d027a9a6e0f33b75daf2bced617b9835 and https://github.com/WordPress/gutenberg/pull/61022/commits/b8768b8261678524282f1fbdd70990f65d2e751e